pulsar-testclient/src/main/java/org/apache/pulsar/testclient/ManagedLedgerWriter.java

```diff
@@ -98,10 +98,16 @@
                 description = "Number of threads writing", validateWith = PositiveNumberParameterValidator.class)
         public int numThreads = 1;
 
-        @Parameter(names = { "-zk", "--zookeeperServers" },
-                description = "ZooKeeper connection string", required = true)
+        @Deprecated
+        @Parameter(names = {"-zk", "--zookeeperServers"},
+                description = "ZooKeeper connection string",
+                hidden = true)
         public String zookeeperServers;
 
+        @Parameter(names = {"-md",
+                "--metadata-store"}, description = "Metadata store service url. eg: zk:my-zk:2181")
```

Comment:
```suggestion
                "--metadata-store"}, description = "Metadata store service URL. For example: zk:my-zk:2181")
```

Please capitalize proper nouns correctly. Details: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.je5thtkj42cb